### ISSUE TITLE
Twilight is overriding custom theme again

### ISSUE THREAD

**Issue by mallowigi** [StylishThemes/GitHub-Dark#1077]
It happened pretty recently, somehow there are two themes loaded in the webpage, the selected one plus Twilight. Apparently the `remove start` wasn't executed

**Comment by the-j0k3r** [comment]
What webpage? Thers not much to do with the information provided.

**Comment by mallowigi** [comment]
Ok let me rephrase it:

Since very recently, the custom theme in the source code (CodeMirror) has become the Twilight theme, independently of the theme selected (currently One Dark). I've inspected the style tag that was injected in the webpage (Github) and I see that the section that was usually removed by the grunt task, encapsulated by `remove start` and `remove end` comments, is currently there, which is causing the Twilight theme to override the theme.

**Comment by the-j0k3r** [comment]
Again what web page? In order to fix I must see the issue.

**Comment by the-j0k3r** [comment]
I cant duplicate any issues, are you sure you are using this GitHub Dark and not some chrome extension?

**Comment by mallowigi** [comment]
I'm not using Stylus but the TamperMonkey script. 

I've reinstalled from scratch to make sure

Here's my settings window. I've changed the codeMirror theme to Github Dark
<img width="210" alt="window" src="https://user-images.githubusercontent.com/5015756/75668699-64828280-5c82-11ea-8094-10a95b110d9d.png">
But here's the code: Twilight theme
<img width="213" alt="code" src="https://user-images.githubusercontent.com/5015756/75668705-68160980-5c82-11ea-9d60-77a6821bb2db.png">

Thank you for your patience.

**Comment by the-j0k3r** [comment]
Please use stylus + Github Dark usercss that script is buggy and none is really maintaining it or fixing the big issues.

**Comment by mallowigi** [comment]
Ok I’ll post there but I wish someone will take care of it, I prefer using the script as it is easier to change the accent color or the themes.

In the worst case scenario I’ll fix it myself like I did with another script that I use every day.

**Comment by the-j0k3r** [comment]
>I prefer using the script as it is easier to change the accent color or the themes.

Its really easy with GitHub Dark + Stylus

![generic](https://user-images.githubusercontent.com/31389848/75679673-5f2c3480-5c90-11ea-8298-f89ebc5895ef.gif)

Also The script cant handle multiple @moz-document rules so it will be broken by default.

**Comment by the-j0k3r** [comment]
Be advised that grunt was removed from this style, Github Dark script loads this style. so IDK.

**Comment by mallowigi** [comment]
Alright, you can close ths issue
